protos/bottle/inventory/events/v1/component_availability_requested.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+import "bottle/inventory/v1/component.proto";
+
+package bottle.inventory.events.v1;
+
+message ComponentAvailabilityRequested {
```

Comment:
Having a quantity request and change in queue seems weird to me, but I guess this makes sense with our current infrastructure. I know that aws elb support grpc if we want to start working with that for this project.

Reply:
This has felt meh to me as well. My thinking was they don't need to be synchronous so it didn't seem necessary to introduce a mechanism to achieve that when it isn't required.

(We discussed in Slack, we'll use this for now and circle back to this later)